### PR TITLE
Changes to the case feature in 8.7 

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Fixes https://github.com/elastic/security-docs/issues/2799.

Previews:
- [Overview dashboard](https://security-docs_3053.docs-preview.app.elstc.co/guide/en/security/master/overview-dashboard.html) - Refreshed both screenshots in the topic. 
- [Cases](https://security-docs_3053.docs-preview.app.elstc.co/guide/en/security/master/cases-overview.html) - Refreshed the topic.
- [Copy the case UUID](https://security-docs_3053.docs-preview.app.elstc.co/guide/en/security/master/cases-open-manage.html#cases-copy-case-uuid) - New section.
- [Add alerts to an existing case](https://security-docs_3053.docs-preview.app.elstc.co/guide/en/security/master/signals-to-cases.html#signals-to-existing-cases) - Refreshed the screenshot.

### CODE REVIEW

File: docs/cases/cases-manage.asciidoc [hunk 1]
```diff
@@ -134,6 +135,15 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
 
+[float]
+[[cases-copy-case-uuid]]
+=== Copy a case's UUID
```
Comment (nastasha-solomon) [inline docs/cases/cases-manage.asciidoc:140]:
```suggestion
=== Copy the case UUID
```

**Review by jonathan-buttner — APPROVED** [review]

File: docs/cases/cases-manage.asciidoc [hunk 2]
```diff
@@ -134,6 +135,15 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
 
+[float]
+[[cases-copy-case-uuid]]
+=== Copy the case UUID
+
+Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and find the case you want to share, click the *Actions* menu, then select *Copy Case ID*. Alternatively, go to a case's details page, click the *More actions* menu (…​), and select *Copy Case ID*.
```
Comment (joepeeples) [inline docs/cases/cases-manage.asciidoc:142]: Optional streamlining:
```suggestion
Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then select *More actions* menu (…​) -> *Copy Case ID*.
```

File: docs/cases/cases-manage.asciidoc [hunk 3]
```diff
@@ -134,6 +135,15 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
...
+
+[role="screenshot"]
+image::images/cases-copy-case-id.png[Shows where the Copy Case ID option is 60%,60%]
```
Comment (joepeeples) [inline docs/cases/cases-manage.asciidoc:145]:
```suggestion
image::images/cases-copy-case-id.png[Copy Case ID option in More actions menu 60%,60%]
```

File: docs/detections/images/add-alert-to-existing-case.png
Comment (joepeeples) [inline docs/detections/images/add-alert-to-existing-case.png:1]: Any chance you could re-crop to just include the modal? The gray background looks like a really thick border

**Review by joepeeples — COMMENTED** [review]
LGTM, just a couple minor suggestions.

**Review by benironside — APPROVED** [review]
LGTM!

File: docs/cases/cases-manage.asciidoc [hunk 4]
```diff
@@ -134,6 +135,15 @@ NOTE: Once a visualization has been added to a case, it cannot be modified or de
 [role="screenshot"]
 image::images/cases-open-vis.png[Shows where the Open Visualization option is]
 
+[float]
+[[cases-copy-case-uuid]]
+=== Copy the case UUID
+
+Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then select *More actions* menu (…​) -> *Copy Case ID*.
```
Comment (jmikell821) [inline docs/cases/cases-manage.asciidoc:142]:
```suggestion
Each case has a universally unique identifier (UUID) that you can copy and share. To copy a case's UUID to a clipboard, go to the Cases page and select *Actions* -> *Copy Case ID* for the case you want to share. Alternatively, go to a case's details page, then from the *More actions* menu (…​), select *Copy Case ID*.
```

**Review by jmikell821 — COMMENTED** [review]
Very tiny UX nit, then good to go, thanks!

**Pull request merged**